config/menus/options.cfg

```diff
@@ -270,11 +270,14 @@ options_console = [
         ]
     ]
 
-    ui_font "little" [ ui_text "^faDefault: 32" ]
+    ui_font "little" [ ui_text "^faDefault: 32" ]; ui_strut 0.5
 
-    ui_strut 0.5
+    ui_checkbox "Play sound when mentioned in chat" playmentionedsound; ui_strut 0.5
 
-    ui_checkbox "Play sound when mentioned in chat" playmentionedsound
+    ui_list [
+        ui_checkbox "Hide Chat" hidechat; ui_strut 0.5
+        ui_checkbox "Mute Chat" mutechat
+    ]
```

Comment:
These need an explanation of what the three-state checkboxes do.

Reply:
added